app/views/dashboards/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1 class="t-display page__heading">Dashboard</h1>
+<% @title = t('.title') %>
```

Comment:
@sonalkr132 Thanks for your :eyes: ! I've not removed the heading.

Please check [app/views/layouts/application.html.erb#L106](https://github.com/rubygems/rubygems.org/blob/master/app/views/layouts/application.html.erb#L106)

```erb
          <% if @title %>
            <h1 class="t-display page__heading">
              <%= link_to @title, @title_url, class: "t-link--black" %>

              <% if @subtitle %>
                <i class="page__subheading"><%= @subtitle.html_safe %></i>
              <% end %>
            </h1>
          <% end %>
```